src/coreComponents/physicsSolvers/solidMechanics/SolidMechanicsLagrangianFEM.hpp

```diff
@@ -291,6 +296,7 @@ class SolidMechanicsLagrangianFEM : public SolverBase
   real64 m_maxForce = 0.0;
   integer m_maxNumResolves;
   integer m_strainTheory;
+  integer m_pressureEffectsFlag;
```

Comment:
I don't love this. It's hard to understand what it means for a general `solidMechanicsSolver`.

Reply:
That is also true, but it reduces a lot of code compared to using a full porousSolid material, and having to switch off the flow solvers.